src/board/m5stack/M5CORE2.h

```diff
@@ -0,0 +1,244 @@
+/*
+ * SPDX-FileCopyrightText: 2023-2024 Espressif Systems (Shanghai) CO LTD
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#pragma once
+
+// *INDENT-OFF*
+
+////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////
+//////////////////////////// Please update the following macros to configure the LCD panel /////////////////////////////
+////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////
+/* Set to 1 when using an LCD panel */
+#define ESP_PANEL_USE_LCD           (1)     // 0/1
+
+#if ESP_PANEL_USE_LCD
+/**
+ * LCD Controller Name.
+ * LCD Controller of M5Core2 is ILI9342C, but the driver is compatible with ILI9341.
+ */
+#define ESP_PANEL_LCD_NAME          ILI9341
+
+/* LCD resolution in pixels */
+#define ESP_PANEL_LCD_WIDTH         (320)
+#define ESP_PANEL_LCD_HEIGHT        (240)
+
+/* LCD Bus Settings */
+/**
+ * If set to 1, the bus will skip to initialize the corresponding host. Users need to initialize the host in advance.
+ * It is useful if other devices use the same host. Please ensure that the host is initialized only once.
+ */
+#define ESP_PANEL_LCD_BUS_SKIP_INIT_HOST    (0)     // 0/1
+/**
+ * LCD Bus Type.
+ */
+#define ESP_PANEL_LCD_BUS_TYPE      (ESP_PANEL_BUS_TYPE_SPI)
+/**
+ * LCD Bus Parameters.
+ *
+ * Please refer to https://docs.espressif.com/projects/esp-idf/en/latest/esp32s3/api-reference/peripherals/lcd.html and
+ * https://docs.espressif.com/projects/esp-iot-solution/en/latest/display/lcd/index.html for more details.
+ *
+ */
+#if ESP_PANEL_LCD_BUS_TYPE == ESP_PANEL_BUS_TYPE_SPI
+
+    #define ESP_PANEL_LCD_BUS_HOST_ID           (1)     // Typically set to 1
+    #define ESP_PANEL_LCD_SPI_IO_CS             (5)
+#if !ESP_PANEL_LCD_BUS_SKIP_INIT_HOST
+    #define ESP_PANEL_LCD_SPI_IO_SCK            (18)
+    #define ESP_PANEL_LCD_SPI_IO_MOSI           (23)
+    #define ESP_PANEL_LCD_SPI_IO_MISO           (38)    // -1 if not used
```

Comment:
Just to confirm, does this pin is truely used?

Reply:
You are really attentive; The MISO on this board is indeed not used. I have already pushed the modifications up.